lib/components/map/point-popup.tsx

```diff
@@ -1,45 +1,68 @@
 import { connect } from 'react-redux'
-import { Popup, useMap } from 'react-map-gl'
+import { Popup as MapGlPopup, MapRef, useMap } from 'react-map-gl'
+import { Search } from '@styled-icons/fa-solid/Search'
 import { useIntl, WrappedComponentProps } from 'react-intl'
 import FromToLocationPicker from '@opentripplanner/from-to-location-picker'
-import React, { useEffect } from 'react'
+import React, { useCallback } from 'react'
 import styled from 'styled-components'
 import type { Location } from '@opentripplanner/types'
 
 import * as mapActions from '../../actions/map'
+import { Icon } from '../util/styledIcon'
 import { renderCoordinates } from '../form/user-settings'
 
-const PopupContainer = styled.div`
-  width: 240px;
+/**
+ * Adds a box shadow and tweaks border radius to make popups easier to read.
+ */
+const Popup = styled(MapGlPopup)`
+  & > .maplibregl-popup-content,
+  & > .mapboxgl-popup-content {
+    border-radius: 6px;
+    box-shadow: 0 3px 14px rgb(0 0 0 / 40%);
+  }
```

Comment:
<img width="452" alt="Screen Shot 2022-10-20 at 2 02 42 PM" src="https://user-images.githubusercontent.com/86619099/197024235-8721d1de-520d-4026-8d10-72564d06882f.png">
I love these changes, but would it be possible to extract this into core-utils? That way the popups can be a bit more consistent.

Reply:
They would be in the map package, but I get the idea and agree. Should that be a `feat.` or a `fix` change?

Reply:
fix I think since it's mostly cosmetic

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/667/commits/3e590c94392954fd6870e1895e5205d82b72a9f1.